### PR TITLE
Prep amazon.aws 5.5.3

### PR DESCRIPTION
**Pull request by hakbailey**
##### SUMMARY
Prep 5.5.3 release.

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,20 +5,34 @@ amazon.aws Release Notes
...
 ---------------
 
-This release brings few bugfixes.
+This release brings a few bugfixes.
 
 Bugfixes
 --------
 
 - cloudwatchevent_rule - Fixes changed status to report False when no change has been made. The module had incorrectly always reported a change. (https://github.com/ansible-collections/amazon.aws/pull/1589)
-- lambda_execute - Fixes to the stack trace output, where it does not contain spaces between each character. The module had incorrectly always outputted extra spaces between each character. (https://github.com/ansible-collections/amazon.aws/pull/1615)
```
Comment (alinabuzachis) [inline CHANGELOG.rst:20]: Why these changes?

Comment (hakbailey) [reply CHANGELOG.rst:20]: That happened automatically when I ran antsibull-changelog, I assumed it was putting the bugfixes in alphabetical order?

Comment (tremble) [reply CHANGELOG.rst:20]: @felixfontein Any idea why antsibull-changelog would change the order of an old changelog entry?

Comment (tremble) [reply CHANGELOG.rst:20]: Oh, I see something weird happened with the release note for the previous release in changelog.yaml, so it considered 5.5.2 fair game for reformatting.

**Review by tremble — APPROVED** [review]
It looks like something might have gone slightly sideways with the previous release.  If I had to guess, I'd suspect a local change that didn't get pushed with the last release.

*Please beware*:
antsibull-changelog will read the original fragments, add them into changelog.yaml, and then generate CHANGELOG.rst from changelog.yaml.  Any tweaks needed to CHANGELOG.rst should first be made in changelog.yaml, and you can then run `antsibull-changelog generate` to regenerate CHANGELOG.rst.

**Review by alinabuzachis — APPROVED** [review]

**Review by tremble — APPROVED** [review]
